### ISSUE TITLE
MySQL doc should also list MariaDB

### ISSUE THREAD

**Issue by alsargent** [influxdata/docs-v2#783]
_Describe the issue here._

The doc/github page https://github.com/influxdata/telegraf/tree/master/plugins/inputs/mysql doesn't have any mention that this plugin works for MariaDB. It does work (@ssoroka tested it last night), so please say so on this page.

##### Relevant URLs
https://github.com/influxdata/telegraf/tree/master/plugins/inputs/mysql

**Comment by noramullen1** [comment]
Thanks for the issue @alsargent . My understanding is that the Telegraf team maintains the GitHub documentation for plugins while we on the docs team just do the versions on the docs site (https://v2.docs.influxdata.com/v2.0/reference/telegraf-plugins/). @sanderson and @danielnelson , can you confirm?

**Comment by danielnelson** [comment]
Yes, any issues with the docs in the telegraf repository should be opened over there.  I'll transfer this issue.  Of course would love any contributions from the docs team on the telegraf repo.
